Fetch/Fetch.php

```diff
@@ -301,26 +327,6 @@ private function filterData()
         return $this;
     }
 
-    /**
-     * Filter unpublished content
-     */
-    private function filterUnpublished($data)
```

Comment:
These are unused so I deleted them

Reply:
Actually they get called dynamically 😬 

See here: https://github.com/aryehraber/statamic-fetch/blob/master/Fetch/Fetch.php#L282:L302

Reply:
haha I figured that was the case. 

Tbh, I'm not sure this method needs to be as fancy (complex) as it is... I think I was initially planning on having more filter types so this way they would easily be added without extra logic. Might refactor this at some point, but for now thanks for restoring!